backend/ultra/settings.py

```
@@ -192,6 +192,50 @@
+            'propagate': False,
+        },
+        'django': {
+            'handlers': ['console'],
+            'level': 'INFO',
+            'propagate': False,
+        },
+    },
+}
+
+# During tests, suppress validation/integrity error logging and request warnings
+import sys
```

Ruff (0.14.2)

Module level import not at top of file

(E402)